woocommerce/woocommerce
# Split the Extensions page in WP Admin and add respective menu items

Continuation of #30378 so that phpcs errors on the My Subscriptions page aren't a blocker for now.

### All Submissions:






### Changes proposed in this Pull Request:
- Register submenu for My Subscriptions under WooCommerce
- Add a method in `WC_Admin_Addons` for Helper page output only
- remove top nav from the Extensions page (keeping it on the My Subscriptions page for now so phpcs is not a blocker)
- update e2e tests for Helper connection.


### How to test the changes in this Pull Request:

1. Check out this branch on local environment.
2. Go to WP Admin and observe there are two separate submenus for Extensions and My Subscriptions.
3. Open each of these pages and make sure they load correctly and have all styles in place.
4. Make sure the correct menu item is highlighted when you open each page.

### Other information:


### Screenshots
![](https://d.pr/i/55j3Qb+)

### Notes
- didn't remove top nav on the My Subscriptions page yet because editing this file triggered a lot of phpcs errors. I'll take care of it later.
